src/pages/components/tabs/usage.mdx

```diff
@@ -210,19 +205,34 @@ A. Label <br /> B. Indicator <br /> C. Scroll button
 
 #### 3. Contained tabs with icon and secondary label
 
-A. Label <br /> B. Indicator <br /> D. Icon (optional) <br /> E. Container<br />
-F. Secondary label (optional) <br /> G. Tab panel
+A. Label <br /> B. Indicator <br /> D. Icon (optional) <br /> E. Tab panel<br />
+F. Secondary label (optional)
 
 </Column>
 <Column colSm={2} colMd={4} colLg={4}>
 
 #### 2. Icon-only contained tabs
 
-B. Indicator <br /> D. Icon <br /> E. Container
+B. Indicator <br /> D. Icon
 
 </Column>
 </Row>
 
+### Anatomy of Vertical tabs
```

Comment:
Is the "Vertical tab" supposed to be capitalized? or it could be " Anatomy of vertical tabs"?